### PR TITLE
Add profiles for the new GV100 GPUs on Hemera

### PR DESCRIPTION
**Pull request by BeyondEspresso**
This commit adds the profiles for the new V100-GPUs of the CASUS cluster. The profiles are very similar to the `gpu`-queue.

### CODE REVIEW

**Comment by BeyondEspresso** [comment]
@sbastrakov Ok, I added the documentation.

**Comment by steindev** [comment]
Is there a reason you set  `PIC_BACKEND` to `cuda:60` instead of `cuda:70`? See (https://developer.nvidia.com/cuda-gpus#compute)

**Comment by BeyondEspresso** [comment]
No, there was no reason. Thanks for catching this.

**Comment by steindev** [comment]
And the standard .tpl file name should be gv100.tpl in TBG_TPLFILE="etc/picongpu/hemera-hzdr/v100.tpl".

Both in the new etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example

**Comment by BeyondEspresso** [comment]
Thanks, I included @steindev 's feedback.

File: etc/picongpu/hemera-hzdr/gv100.tpl [hunk 1]
```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```
Comment (psychocoderHPC) [inline etc/picongpu/hemera-hzdr/gv100.tpl:1]: Please rename this file to `fwkt_v100.tpl`. Our files have always the name of the queue.

File: etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Name and Path of this Script ############################### (DO NOT change!)
```
Comment (psychocoderHPC) [inline etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example:1]: please name this file to `fwkt_v100_picongpu.profile.example`

**Comment by psychocoderHPC** [comment]
> And the standard .tpl file name should be `gv100.tpl` in `TBG_TPLFILE="etc/picongpu/hemera-hzdr/v100.tpl"`.
> 
> Both found in the new `etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example`

We call the files always with the queue name as prefix.

File: etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+export PATH=$PATH:$PICSRC/bin
+export PATH=$PATH:$PICSRC/src/tools/bin
+
+export PYTHONPATH=$PICSRC/lib/python:$PYTHONPATH
+
+# "tbg" default options #######################################################
+#   - SLURM (sbatch)
+#   - "gpu" queue
+export TBG_SUBMIT="sbatch"
+export TBG_TPLFILE="etc/picongpu/hemera-hzdr/gv100.tpl"
```
Comment (psychocoderHPC) [inline etc/picongpu/hemera-hzdr/gv100_picongpu.profile.example:55]: do not forget to change this filename too

**Comment by BeyondEspresso** [comment]
@psychocoderHPC Thanks, I included your feedback.

**Comment by PrometheusPi** [comment]
The Travis-CI fails repeatedly with a pyflakes issue on a plotting script @BeyondEspresso Did not change.

File: docs/source/install/profile.rst [hunk 4]
```diff
@@ -41,6 +41,12 @@ Queue: gpu (4x NVIDIA P100 16GB)
 .. literalinclude:: profiles/hemera-hzdr/gpu_picongpu.profile.example
    :language: bash
 
+Queue: fwkt_v100 (4x NVIDIA V100 32GB)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: profiles/hemera-hzdr/gv100_picongpu.profile.example
```
Comment (psychocoderHPC) [inline docs/source/install/profile.rst:47]: please update the file name to: `profiles/hemera-hzdr/fwkt_v100_picongpu.profile.example`

**Comment by psychocoderHPC** [comment]
@BeyondEspresso please also squash you commits to one.

**Comment by psychocoderHPC** [comment]
> The Travis-CI fails repeatedly with a pyflakes issue on a plotting script @BeyondEspresso did not change.

The python issue should be fixed with #3232

**Comment by PrometheusPi** [comment]
@BeyondEspresso #3232 was merged to `dev`- the Travis-CI should pass if you update your branch.

**Comment by BeyondEspresso** [comment]
@psychocoderHPC I rebased to the updated dev and squashed my commits.

**Pull request merged**